File: problem2/problem.js
function fiboEvenSum(n){
    let fibonacci = []
    let sum = 0
    for (let index = 0; index < n+2; index++) {
        if(index == 0){
            fibonacci.push(0)
        }else if(index == 1){
            fibonacci.push(1)
        }else{
            fibonacci.push(fibonacci[index-2] + fibonacci[index-1])
        }
    }
    fibonacci.forEach(element => {
        if (element%2 == 0){
            sum = sum + element
        }
    });
    return sum;
}

console.log(fiboEvenSum(10))
console.log(fiboEvenSum(18))
console.log(fiboEvenSum(23))
console.log(fiboEvenSum(43))

File: problem2/problem.py
def fiboEvenSum(n):
    fibonacci = []
    for i,j in enumerate(range(0,n+2)):
        if j == 0:
            fibonacci.append(0)
        elif j == 1:
            fibonacci.append(1)
        else:
            fibonacci.append(fibonacci[i-2]+fibonacci[i-1])
    return sum([x for x in fibonacci if x%2==0])